cluster-up/cluster/kind/common.sh

```diff
@@ -39,7 +39,13 @@ ETCD_IN_MEMORY_DATA_DIR="/tmp/kind-cluster-etcd"
 
 function _wait_kind_up {
     echo "Waiting for kind to be ready ..."
-    while [ -z "$(docker exec --privileged ${CLUSTER_NAME}-control-plane kubectl --kubeconfig=/etc/kubernetes/admin.conf get nodes --selector=node-role.kubernetes.io/master -o=jsonpath='{.items..status.conditions[-1:].status}' | grep True)" ]; do
+    # For providers with version > 1.19 we use control-plane node role, master otherwise
+    if [[ $KUBEVIRT_PROVIDER =~ k8s-1\.1.* ]] || [[ $KUBEVIRT_PROVIDER =~ kind-1\.1.* ]]; then
```

Comment:
this would work `if [[ $KUBEVIRT_PROVIDER =~ (k8s|kind)-1\.1.* ]]; then` i think as well
but on the other hand, i think we should not limit kind to specific versions, so maybe lets
change please `kind-1\.1.*` to simply `kind-.*` ?
so once we bump kind it won't stop working, because what matters here is that it is kind, not the version for now

Reply:
@oshoval Thanks so much for revision. Maybe I'm missing something, but I think that version is relevant. Only kind with version <1.20 doesn't have `control-plane` label. Indeed, in future k8s versions (and kind too) `master` will not be available anymore. In fact kind with version 1.22 works with `control-plane` selector. Anyway you are right about `if [[ $KUBEVIRT_PROVIDER =~ (k8s|kind)-1\.1.* ]]; then`. If you agree I will change the if with the latter, otherwise can you explain why version is not important? :slightly_smiling_face:  Thanks

Reply:
ah ok you are right then, thanks

Reply:
if so, you are right, but
i am just confused with one thing, we already have kind-1.22-sriov
and it still using master

another note, this function _wait_kind_up is only for kind, so you can remove the k8s regex i think ?

Reply:
`control-plane` label is added in parallel of `master` by kubeadm itself. In this co-living period it can be used master and control-plane indiscriminately. So maybe for the next providers I think that is better to use always `control-plane`. About the regex I left the k8s part because I see that there is a provider named `kind-k8s-1.19`, but we are not using it, so maybe I was too conservative. I will remove it. Thanks

Reply:
people from `arm` do use kind-k8s-1.19 indeed, but having `kind-.*` will include that as well
thanks

Reply:
If you think that something like `kind-k8s-1.21.1` should never happen ok, otherwise we can switch to `kind-(k8s-)?1\.1.*`

Reply:
we don't specify the Z stream as part of the providers
(since we bump the same provider with Z stream, but the major and minor are pinned)

Reply:
Perfect, I will use `kind-.*1\.1.*`. Thanks for information and support :slightly_smiling_face: